hydroflow_plus_test/src/cluster/simple_cluster.rs

```diff
@@ -30,6 +52,51 @@ mod tests {
     use hydro_deploy::Deployment;
     use hydroflow_plus::deploy::{DeployCrateWrapper, TrybuildHost};
 
+    #[tokio::test]
+    async fn decouple_cluster() {
+        let mut deployment = Deployment::new();
+
+        let builder = hydroflow_plus::FlowBuilder::new();
+        let (cluster1, cluster2) = super::decouple_cluster(&builder);
+        let built = builder.with_default_optimize();
+
+        let nodes = built
+            .with_cluster(
+                &cluster1,
+                (0..3)
+                    .map(|_| TrybuildHost::new(deployment.Localhost()))
+                    .collect::<Vec<_>>(),
+            )
+            .with_cluster(
+                &cluster2,
+                (0..3)
+                    .map(|_| TrybuildHost::new(deployment.Localhost()))
+                    .collect::<Vec<_>>(),
+            )
+            .deploy(&mut deployment);
+
+        deployment.deploy().await.unwrap();
+
+        deployment.start().await.unwrap();
```

Comment:
This unit test doesn't actually check anything; we should be inspecting the `stdout` of nodes.